services/payments/statemachine_bitflyer.go

```diff
@@ -0,0 +1,48 @@
+package payments
+
+import (
+	"context"
+	"errors"
+
+	"github.com/brave-intl/bat-go/libs/clients/bitflyer"
+)
+
+// DriveBitflyerTransaction returns a new, validly progressed transaction state.
+func DriveBitflyerTransaction(
+	ctx context.Context,
+	currentTransactionState QLDBPaymentTransitionHistoryEntry,
+	transactionSet bitflyer.WithdrawToDepositIDBulkPayload,
+) (QLDBPaymentTransitionState, error) {
+	switch currentTransactionState.Data.Status {
+	case 0:
```

Comment:
If these are states the tx can be in, lets use the State consts to make this more clear